src/test/java/org/folio/rest/impl/PurchaseOrdersApiTest.java

```diff
@@ -417,7 +417,6 @@ public void testListedPrintMonographInOpenStatus() throws Exception {
     // Check that search for existing instances was done not for all PO lines
     assertEquals(polCount - 1, instancesSearches.size());
 
-    verifyInventoryInteraction(resp, polCount);
```

Comment:
I think we still need to test InventoryInteraction every time we create open order or update workflowStatus from Pending to Open.

Reply:
Agree, done